tests/core.py

```diff
@@ -1696,6 +1696,15 @@ def test_delete_dag(self):
                 '--yes'])
         )
 
+        # Test to check that the DAG should be deleted even if
+        # the file containing it is not deleted
```

Comment:
I think it would be better to just create a new test.
`def test_delete_existing_dag(self)` or `def test_delete_dag_exists(self)`

Reply:
Done